integration/commands_administration_test.go

```diff
@@ -574,6 +574,29 @@ func TestCommandsAdministrationBuildInfo(t *testing.T) {
 	assert.True(t, ok)
 }
 
+func TestCommandsAdministrationBuildInfoFerretdbExtensions(t *testing.T) {
+	setup.SkipForMongoDB(t, "FerretDB-specific command's extensions")
+
+	t.Parallel()
+	ctx, collection := setup.Setup(t)
+
+	var actual bson.D
+	command := bson.D{{"buildInfo", int32(1)}}
+	err := collection.Database().RunCommand(ctx, command).Decode(&actual)
+	require.NoError(t, err)
+
+	doc := ConvertDocument(t, actual)
+
+	// ferretdbFeatures, err := doc.Get("ferretdbFeatures").(*types.Document)
+	ferretdbFeatures, err := doc.Get("ferretdbFeatures")
+	assert.NoError(t, err)
+	ferretdbFeaturesDoc := ferretdbFeatures.(*types.Document)
+	assert.NotNil(t, ferretdbFeatures)
+	aggregationStages, err := ferretdbFeaturesDoc.Get("aggregationStages")
+	assert.NoError(t, err)
+	assert.NotEmpty(t, aggregationStages.(*types.Array))
```

Comment:
Let's also assert aggregationStages is an array, `stages, ok := aggregationStages.(*types.Array)` and assert that ok is true to avoid panic in test :hugs: